src/hotspot/share/vitals/vitals.cpp

```diff
@@ -752,14 +752,14 @@ class PrintSamplesClosure : public SampleTable::Closure {
   }
 };
 
-// sampleTables is a combination of three tables: a short term table, a mid term table, a long term table.
```

Comment:
nit: a short term table and a long term table

Reply:
done